app.test.js

```diff
@@ -94,4 +94,34 @@ describe("GET /api/articles/:article_id", () => {
             expect(body.message).toBe("invalid id path")
         })
     })
+})
+
+
+describe("GET /api/articles", () => {
+    test("returns all articles on this endpoint", () => {
+        return request(app)
+        .get("/api/articles")
+        .expect(200)
+        .then(({body}) => {
+            let resBody = body.response
+
+            expect(Object.keys(resBody[2])).toHaveLength(8)
+            expect(resBody)
+
+            expect(resBody[7]).toEqual(
+                expect.objectContaining({
+                    article_id: expect.any(Number),
+                    title: expect.any(String),
+                    topic: expect.any(String),
+                    author: expect.any(String),
+                    created_at: expect.any(String),
+                    votes: expect.any(Number),
+                    article_img_url: expect.any(String),
+                    comment_count: expect.any(String)
```

Comment:
Ideally the comment count would be a number rather than a string
You could potentially achieve this using casting https://www.w3schools.com/sql/func_sqlserver_cast.asp